### PR TITLE
[Router] Add recipe for 5.1 router default URI config

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | N/A    

Follows https://github.com/symfony/symfony/pull/35281

### CODE REVIEW

File: symfony/routing/5.1/config/packages/routing.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+framework:
+    router:
+        utf8: true
+
+        # Replace these node to configure the "request context" to use to generate URLs
+        # in non-HTTP contexts, such as CLI commands.
+        # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
```
Comment (ogizanagi) [inline symfony/routing/5.1/config/packages/routing.yaml]: This doc is not up-to-date yet, but there is probably no reason to move it elsewhere.

File: symfony/routing/5.1/config/packages/routing.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+framework:
+    router:
+        utf8: true
+
+        # Replace these nodes to configure the "request context" to use to generate URLs
```
Comment (nicolas-grekas) [inline symfony/routing/5.1/config/packages/routing.yaml]: Configure how to generate URLs in non-HTTP contexts, such as CLI commands.

File: symfony/routing/5.1/config/packages/routing.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        # in non-HTTP contexts, such as CLI commands.
+        # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
+        #context:
+        #    host: localhost
+        #    scheme: http
+        #    base_url: null
```
Comment (nicolas-grekas) [inline symfony/routing/5.1/config/packages/routing.yaml]: shouldn't this be a single setting: `http://lcalhost/path/`?

File: symfony/routing/5.1/config/packages/routing.yaml (hunk 2 above)
Comment (ogizanagi) [reply symfony/routing/5.1/config/packages/routing.yaml]: Shorter is better, thx.

File: symfony/routing/5.1/config/packages/routing.yaml (hunk 3 above)
Comment (ogizanagi) [reply symfony/routing/5.1/config/packages/routing.yaml]: That's not how the router context works right now. Do you thinks it's worth adding such support?

Comment (nicolas-grekas) [reply symfony/routing/5.1/config/packages/routing.yaml]: I think yes, that's how it should work to me.

Comment (ogizanagi) [reply symfony/routing/5.1/config/packages/routing.yaml]: @benji07: interested?

Comment (nicolas-grekas) [reply symfony/routing/5.1/config/packages/routing.yaml:10]: Tracked in https://github.com/symfony/symfony/issues/35121, implementation proposed in https://github.com/symfony/symfony/pull/35580

**Comment by nicolas-grekas** [comment]
To be updated for https://github.com/symfony/symfony/pull/36651

**Comment by fabpot** [comment]
Upstream PR has been merged now.

**Comment by ogizanagi** [comment]
@fabpot @nicolas-grekas PR updated

**Pull request merged**